### PR TITLE
30 authentication tests in usercontroller

### PR DESCRIPTION
**Pull request by desmeach**
Authentication cases added into UserControllerTest

### CODE REVIEW

**Review by l3r8yJ — CHANGES_REQUESTED** [review]
try to fix **all** methods names into UserControllerTests, please

File: src/test/java/ru/leroy/screenerapi/controller/UserControllerTest.java [hunk 1]
```diff
@@ -72,6 +74,71 @@ void registration_success() throws Exception {
...
+                                .contentType(MediaType.APPLICATION_JSON)
+                )
+                .andReturn()
+                .getResponse();
+        assertThat(response.getStatus())
+                .isEqualTo(HttpStatus.UNAUTHORIZED.value());
+    }
+
+    @Test
+    void authentication_emailNotFound() throws Exception {
```
[the remark this answers is not in the document]
Comment (desmeach) [reply src/test/java/ru/leroy/screenerapi/controller/UserControllerTest.java]: done

File: src/test/java/ru/leroy/screenerapi/controller/UserControllerTest.java [hunk 2]
```diff
@@ -72,6 +74,71 @@ void registration_success() throws Exception {
...
+                .andReturn()
+                .getResponse();
+        assertThat(response.getStatus())
+                .isEqualTo(HttpStatus.OK.value());
+        assertThat(response.getContentAsString())
+                .isEqualTo(this.userJson.write(this.userEntity).getJson());
+    }
+
+    @Test
+    void authentication_fail() throws Exception {
```
[the remark this answers is not in the document]
Comment (desmeach) [reply src/test/java/ru/leroy/screenerapi/controller/UserControllerTest.java]: done

File: src/test/java/ru/leroy/screenerapi/controller/UserControllerTest.java [hunk 3]
```diff
@@ -72,6 +74,71 @@ void registration_success() throws Exception {
         );
     }
 
+    @Test
+    void authentication_success() throws Exception {
```
[the remark this answers is not in the document]
Comment (desmeach) [reply src/test/java/ru/leroy/screenerapi/controller/UserControllerTest.java]: done

**Review by l3r8yJ — APPROVED** [review]
nice

**Pull request merged**